src/lang/en-US.ts

```diff
@@ -61,6 +61,8 @@ const CommonMessages: ResolvedIntlConfig['messages'] = {
     'terms.derivation': `Derivation`,
     'terms.documentation': `Docs`,
     'terms.entity': `Entity`,
+    'terms.sources': `Sources`,
+    'terms.destinations': `Destinations`,
```

Comment:
Since these are getting added can you use this down in `routeTitle.captures` and `routeTitle.materializations`